.github/workflows/release.yml

```diff
@@ -27,6 +27,15 @@ jobs:
         with:
           node-version: lts/*
           check-latest: true
+      - name: Super Linter Code Base
+        uses: github/super-linter@v4
+        env:
+          FILTER_REGEX_INCLUDE: "/src/*"
```

Comment:
FILTER_REGEX_INCLUDE is only considering typescript files. there's no bash script in this pattern.